main.go

```diff
@@ -298,6 +298,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controllers.DatabaseSecretEngineStaticRoleReconciler{
```

Comment:
there should not need to be a change here. Why is this needed?

Reply:
The webhook needs to be in the correct if statement to be controlled by the ENABLE_WEBHOOKS environment variable correctly and the ability to disable webhooks is needed when debugging. the other block was moved to put it in the same section as the other manager setups.